spec/loophp/collection/CollectionSpec.php

```diff
@@ -86,7 +88,7 @@ static function ($item) use (&$stack): bool {
         ];
 
         if ($stack !== $expected) {
-            throw new Exception('The expected value does not match.');
+            throw new MatcherException('The expected value does not match.');
```

Comment:
using the PHPSpec matchers will result in specs being reported as `failed` vs `broken` if the exception is thrown

Reply:
Nice !